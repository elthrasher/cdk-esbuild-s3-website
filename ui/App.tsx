import React from 'react';

const App = (): JSX.Element => (
  <div>
    <h1>esbuild and CDK</h1>
  </div>
);

export default App;
