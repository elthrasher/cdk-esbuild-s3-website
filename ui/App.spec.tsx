import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

it('renders a message', () => {
  const component = renderer.create(<App />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
