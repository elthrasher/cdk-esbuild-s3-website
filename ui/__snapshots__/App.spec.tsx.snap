// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a message 1`] = `
<div>
  <h1>
    esbuild and CDK
  </h1>
</div>
`;
